### PR TITLE
updating engine files: db_storage and file_storage

### PR DESCRIPTION
**Pull request by oscarmrt**
updating engine files: db_storage and file_storage working with def get(self, cls, id) and def count(self, cls=None).

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
...
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Returns the object based on the class name and its ID, or None if not found"""
+        if self.all(cls):
+            for value in self.all(cls).values():
+                if value.id == id:
+                    return value
+        else:
```
Comment (arturovictoriar) [inline models/engine/file_storage.py]: This "else" can be skip

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
...
+            return None
+
+    def count(self, cls=None):
+        """Returns the number of objects in storage matching the given class name.
+        If no name is passed, returns the count of all objects in storage."""
+        if cls:
+            return len(self.all(cls))
+        else:
```
Comment (arturovictoriar) [inline models/engine/file_storage.py]: This "else" can be skip

File: models/engine/db_storage.py [hunk 3]
```diff
@@ -74,3 +74,21 @@ def reload(self):
...
+                    return value
+        else:
+            return None
+
+    def count(self, cls=None):
+        """Returns the number of objects in storage matching the given class name.
+        If no name is passed, returns the count of all objects in storage."""
+        if cls:
+            return len(self.all(cls))
+        else:
```
Comment (arturovictoriar) [inline models/engine/db_storage.py]: This "else" can be skip

File: models/engine/db_storage.py [hunk 4]
```diff
@@ -74,3 +74,21 @@ def reload(self):
...
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class name and its ID,
+        or None if not found"""
+        if self.all(cls):
+            for value in self.all(cls).values():
+                if value.id == id:
+                    return value
+        else:
```
Comment (arturovictoriar) [inline models/engine/db_storage.py]: This "else" can be skip

**Review by arturovictoriar — CHANGES_REQUESTED** [review]

**Review by arturovictoriar — APPROVED** [review]
It is OK to update the master

**Comment by oscarmrt** [comment]
Ok. Thank you.

**Comment by arturovictoriar** [comment]
You are missing the test cases, so you need to add them before i merge the request.

**Review by arturovictoriar — APPROVED** [review]

**Pull request merged**